front/components/assistant_builder/InstructionScreen.tsx

```diff
@@ -462,8 +463,8 @@ function AdvancedSettings({
             </DropdownMenu>
```

Comment:
still there ?

Reply:
Yes, it used to be a `DropdownMenu` opening a `DropdownMenu` 😅 
Now it's a `Popover` containing a `DropdownMenu`